rosdep/python.yaml

```diff
@@ -5437,6 +5437,16 @@ python3-colorama:
   gentoo: [dev-python/colorama]
   rhel: ['python%{python3_pkgversion}-colorama']
   ubuntu: [python3-colorama]
+python3-connexion:
```

Comment:
Since this is a pip-only key, please add the `-pip` suffix to the key, as per [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1).

Reply:
Sorry to have missed that